.travis.yml

```diff
@@ -0,0 +1,46 @@
+# language target
+language: cpp
+compiler: gcc
+
+# main build matrix
+matrix:
+  include:
+    - name: "Bionic Ubuntu Linux"
+      os: linux
+      dist: bionic
+      sudo: required
+
+
+install:
+  - sudo apt update -y
+  # Instal build dependencies
+  - sudo apt install -y build-essential g++ cmake libarchive-dev extra-cmake-modules qtbase5-dev libqt5webkit5-dev libqt5x11extras5-dev libxcb-keysyms1-dev  libsqlite3-dev qtbase5-private-dev xmlto libqt5webkit5-dev libappindicator-dev
+
+  # Install appimage-builder
+  - sudo apt install -y build-essential python3 python3-setuptools python3-pip patchelf gnupg2 file desktop-file-utils fakeroot
+  - sudo pip3 install git+https://github.com/AppImageCrafters/appimage-builder.git
+
+  # Install appimagetool
+  - wget https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage -O /tmp/appimagetool
+  - chmod +x /tmp/appimagetool
+  - sudo mv /tmp/appimagetool /usr/bin
```

Comment:
Why install it globally?

Reply:
The appimagetool binary needs to be found by appimage-builder. Therefore it must be in the PATH. You can deploy it wherever you like as long as you include that path in PATH.

Notice that the downloaded file gets renamed to appimagetool, that's important.